etc/greenboot/motd/redboot.motd

```diff
@@ -1,2 +0,0 @@
-Health Check FAILED! Boot Status is RED!
-Run `journalctl -u greenboot-healthcheck` for more information.
```

Comment:
why not update motd with redboot status?

Reply:
the reason I changed it was that (with auto rollback enabled) you'd never actually see it. Thinking about it a bit more, it probably makes sense to have it, for people not running ostree systems.
Do you think it thus makes sense to split the motd-updater out into its own service unit which is wanted by both greenboot.service and redboot.service units?

Reply:
> Thinking about it a bit more, it probably makes sense to have it, for people not running ostree systems

yeah or people who somehow have an ostree system but don't want "automatic" rollback, but want to investigate first for whatever reason.

> Do you think it thus makes sense to split the motd-updater out into its own service unit which is wanted by both greenboot.service and redboot.service units?

I'm not sure I have an opinion on that. Sounds reasonable to me.